features/support/feature_tests/action/login.rb

```diff
@@ -0,0 +1,33 @@
+module FeatureTests
+  module Action
+    module Login
+      extend Capybara::DSL
+      extend RSpec::Matchers
```

Comment:
These two could be abstracted to a helper module that is then extended in every action/page/section. The helper module would define a Browser constant that is a shortcut to Capybara::DSL module. The helper module would also extend RSpec::Matchers to bring expect, raise_error, eql, etc. into scope.